addons/sys_gsa/XEH_postInit.sqf

```diff
@@ -10,6 +10,12 @@
         _success
     };
 
+    private _connectedUnit = [_radioId] call EFUNC(sys_radio,getRadioObject);
+    if (isNull _connectedUnit) exitWith {
+        ERROR("Null connected object returned");
```

Comment:
This does a pop-up.

Reply:
It was not doing a pop-up if radio is empty...

Reply:
Ah, that may be `ERROR_MSG` then.

Reply:
yes. this only writes ERROR (XXX): in the rpt